scripts/gen_fh.py

```diff
@@ -8,12 +8,11 @@
 # As for numbering, it will take the next number
 # (e.g. if the previous post is FH #1000, the generated one will be FH #1001)
 # Please first update data/friday_hacks.yml before running this
+import re
 import yaml
-from datetime import datetime, timedelta
-from os import listdir
-from os.path import isfile, join
```

Comment:
Will removing this cause any issues?

Reply:
this chunk is actually part of a reordering of imports its not removed

Reply:
Ok!